stable/spotify-docker-gc/README.md

```diff
@@ -0,0 +1,45 @@
+# Spotify's Docker-GC Helm Chart
+
+This chart wraps the [spotify/docker-gc][] Docker image in the form of a [DaemonSet][], so that Docker resource garbage collection occurs on all nodes of a given Kubernetes cluster.
+
+## Chart Details
+This chart will do the following:
+
+* Deploy one pod running the [spotify/docker-gc][] container to each node in a Kubernetes cluster, configured with the values provided.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/spotify-docker-gc
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the Spotify Docker GC chart and their default values.
+
+See the [spotify/docker-gc GitHub repository][] for more settings which may be added to this chart as needed.
+
+| Parameter                         | Description                              | Default                                 |
+| --------------------------------- | ---------------------------------------- | --------------------------------------- |
+| `cron.schedule`                   | cron schedule                            | `@daily`                                |
+| `cron.log`                        | cron log name                            | `/var/logs/cron.log`                    |
+| `env.gracePeriodSeconds`          | grace period in seconds before gc occurs | `0`                                     |
+| `env.dockerAPIVersion`            | Docker API Version for docker-gc client  | Not set                                 |
+| `exclude.images`                  | images to be excluded                    | Not set                                 |
+| `exclude.containers`              | containers to be excluded                | Not set                                 |
+
+
+## Design/Evolution
+
+The initial approach for wrapping [spotify/docker-gc][] in a Helm chart was to use a [CronJob][].  However, as of this writing, there is [no native support][https://github.com/kubernetes/kubernetes/issues/36601] for [DaemonSet][]-like behavior -- that is, ensuring [Job][]s spawned by a given [CronJob][] run on all nodes in a Kubernetes cluster.  Once this feature has been implemented and the [CronJob][] resource itself stabilizes (it is currently in alpha), conversion to this resource would be prudent and will simplify the template/logic.
```

Comment:
The the url in the "no native support" link needs to be wrapped in `()` instead of `[]` to render correctly-- see https://github.com/vdice/charts/blob/06c31c903509289eb164216d35d9220bfeddc959/stable/spotify-docker-gc/README.md#designevolution for current rendering